ci/rbe-image/README.md

```diff
@@ -0,0 +1,41 @@
+# Custom Docker image for running RBE builds
+
+## Purpose
+
+We’ve built a custom RBE image that includes `rpmbuild` which will allow executing `pkg_rpm` on RBE.
+The goal, however, is to keep as close to upstream as possible, so it's built on a stock 'base' image that is used 
+by RBE.
```

Comment:
did you accidentally add newline here?

Reply:
@lolski thanks for noticing! fixed in https://github.com/graknlabs/build-tools/pull/50/commits/6e332f879119bf1cbe87cedc406854471887cecc